theseus/core/robust_cost_function.py

```diff
@@ -82,7 +82,7 @@ def weighted_jacobians_error(self) -> Tuple[List[torch.Tensor], torch.Tensor]:
         ) = self.cost_function.weighted_jacobians_error()
         squared_norm = torch.sum(weighted_error**2, dim=1, keepdim=True)
         loss_radius = torch.exp(self.log_loss_radius.data)
-        rescale = self.loss.linearize(squared_norm, loss_radius)
+        rescale = (self.loss.linearize(squared_norm, loss_radius) + 1e-20).sqrt()
```

Comment:
Does 1E-20 work for `float32` also?

Reply:
`1e-20` works for `float32` in `torch.autograd.functional.jacobian()`. So I guess it should also work in other cases.

Reply:
OK, sounds good, let's merge.